From b95fbd96b79ac8217fa7a04fa4761b17f5edbf45 Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Fri, 10 Nov 2023 16:23:41 +0800
Subject: [PATCH 1/1] fix-install-dir-not-in-usr-local-or-usr-share

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dc7f732..981f6a2 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-PREFIX = /usr/local
+PREFIX = ${DESTDIR}/usr
 
 legacyrenderer:
 	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -DLEGACY_RENDERER:BOOL=true -S . -B ./build -G Ninja
-- 
2.33.1

